### DISCUSSION TITLE
VSCode的變數視窗

### DISCUSSION THREAD

**Discussion by WEI-CHEN-CHUN** [1101-GS4719-C/Python-Community-GS4719-C#800] [General]
請益 
Spyder最好用的地方是變數視窗(variable explorer)
在VS Code 裡頭好像只有Jupyter:Variables
對.py類型的檔案好像不能用
有人有解決方法嗎?

**Comment by pierre0210** [comment]
可以用VS code **Start Debugging** 的功能
![image](https://user-images.githubusercontent.com/54491348/196614585-9e22d7b7-f99f-4a8f-9c2d-ac4fdf87ad53.png)
下break point用step into或step over執行就能看到每一步variable的變化
- step into: 執行的下一步會在呼叫的函式裡
- step over: 將整個函式當作一個步驟執行
